back/Gemfile.lock

```diff
@@ -1313,4 +1322,4 @@ DEPENDENCIES
   whenever (~> 1.0)
 
 BUNDLED WITH
-   2.3.22
+   2.3.20
```

Comment:
Using an older version than before is a step backwards. I would revert this change.

Reply:
Good question :-) Indeed:
```
RUN gem install bundler:2.3.20
```

Reply:
So, which should we use?

Reply:
I'd say `2.3.20` for now if we don't update the `Dockerfile`.

Reply:
OK, I'll revert the reversion.

Reply:
I agree. In the meantime Bundler 2.3.24 exists, but let's update the Dockerfile outside this PR.

Reply:
Done, will now wait for CI (again).